proj-solar-system.rb

```diff
@@ -0,0 +1,101 @@
+
+
+
+class SolarSystem
+
+ def initialize (planets)
+  @planets = planets
+ end
+
+def printPlanet()
+  to_string = ""
+  for i in 0..@planets.length - 1
+    to_string += (i + 1).to_s + ". " + @planets[i].printAttributes +  "\n"
+  end
+ return to_string
+end
+
+ def findPlanetname(planetname)
+   for i in 0..@planets.length - 1
+     planet = @planets[i]
+     if planet.name == planetname
+       return planet
+     end
+   end
+   return nil
+end
+
+def addPlanet(planet)
+  @planets.push(planet)
+end
+
+end
+
+
+class Planet
+  attr_reader :color, :distance, :diameter,:name
+
+  def initialize (color,distance,diameter,name)
+
+    @color = color
+    @distance = distance
+    @diameter = diameter
+    @name = name
+  end
+
+   def summary
+     return "#{@color}: #{@distance}"
+   end
+
+  def printAttributes()
+      to_string = @name + ":" +
```

Comment:
String interpolation might make an easier to read version.  Also the indentation is off